README.md

```diff
@@ -10,16 +10,17 @@
 [![Go Report Card][GoReportImg]][GoReportUrl]
 
 ## Motivation
-Active-Monitor is a Kubernetes custom resource controller which enables deep cluster monitoring using [Argo workflows](https://github.com/argoproj/argo).
+Active-Monitor is a Kubernetes custom resource controller which enables deep cluster monitoring and self-healing using [Argo workflows](https://github.com/argoproj/argo).
 
 While it is not too difficult to know that all entities in a cluster are running individually, it is often quite challenging to know that they can all coordinate with each other as required for successful cluster operation (network connectivity, volume access, etc).
 
 ## Overview
-Active-Monitor will create a new `health` namespace when installed in the cluster. Users can then create and submit HealthCheck objects to the Kubernetes server. A HealthCheck is essentially an instrumented wrapper around an Argo workflow.
+Active-Monitor will create a new `health` namespace when installed in the cluster. Users can then create and submit HealthCheck and Remedy objects to the Kubernetes server. A HealthCheck / Remedy is essentially an instrumented wrapper around an Argo workflow.
```

Comment:
Remedy is not a separate CR now, correct?

Reply:
Yes.. Updated the Readme.